src/app/models/storage-id.model.ts

```diff
@@ -0,0 +1,15 @@
+/**
+ * Used to storage data using StorageService
+ */
+export enum StorageId {
+	/**
+	 * Cached package names that have either been searched
+	 * for or at one point was used to populate the chart
+	 */
+	PACKAGE_NAMES = 'autocomplete-package-names',
```

Comment:
I choose to "rename" these storage ids but not update the actual values to be backward compatible. This would result in a "breaking" change since `autocomplete-package-names` would be `package-names` in local storage.

How do we feel about these renames? Any better suggestions? I feel like these new names make more sense